astropy/io/ascii/html.py

```diff
@@ -263,6 +263,13 @@ class HTML(core.BaseReader):
     In order to customize input and output, a dict of parameters may
     be passed to this class holding specific customizations.
 
+    .. note::
+       Be aware that in many cases reading tables from published HTML journal articles will not work
+       for a variety of reasons, including inconsistent mark-ups, CAPTCHAs, changing formats,
+       embedded javascript, or the table actually being an image. If possible you should consider
+       retrieving the table in a standard data format such as CSV or FITS, perhaps from an archive
+       such as CDS/Vizier.
```

Comment:
perhaps add
```
or download via a "save as data/text file", if provided by the journal, or as next-best option try to open the table in its own tab.
```
Those may still not work in all cases (e.g. using the tab-url was reported to work in https://github.com/astropy/astropy/issues/14346#issuecomment-1410853885, but I could not reproduce it due to permission errors), but could still work as a fallback, if people fail to hunt the data down on Vizier.

Reply:
Most of the "interesting" (meaning long data tables, not the 10 row tables where people report fit results and 3 out of those 10 rows have \multicolumn{} in them) A&A tables are on CDS, and almost none of the AJ/ApJ are. So this is imperfect advice. The question is if this is the right place to essentially write a tutorial for different ways to extract data from an article (I have, for example, also pull the LateX from astro-ph in the past and then read the LaTeX tables). Maybe that would in fact make an interesting tutorial for astropy learn.